content/docs/hooks-reference.md

```diff
@@ -534,14 +534,50 @@ TODO: description
 ### `useId` {#useid}
 
 ```js
-const id = useId(value);
+const id = useId();
 ```
 
-TODO: description
+`useId` is a hook for generating unique IDs that are stable across the server and client, while avoiding hydration mismatches.
+
+For a basic example, simply pass the ID to the elements that need it:
+
+```js
+function Checkbox() {
+  const id = useId();
+  return (
+    <>
+      <label htmlFor={id}>Do you like React?</label>
+      <input id={id} type="checkbox" name="react"/>
+    </>
+  );
+};
+```
+
+For multiple IDs in the same component, `useId` supports appending a suffix:
```

Comment:
If I were reading this, I would infer that useId might not return a normal string, because why else would docs explain that it supports appending?